content/docs/support/cli/_index.md

```diff
@@ -955,22 +956,53 @@ Gets PowerFlex volume infomation for a given tenant on a local cluster
 
 ##### Output
 
-Gets PowerFlex volume infomation for a given tenant on a local cluster. The namespace is the namespace where tenant secret is created. 
+Gets the drivers volume information for a given tenant on a local cluster. The namespace is the namespace where tenant secret is created.
 
 >Note: This was output was generated using Authorization Proxy version 1.5.1. Please ensure you are using version 1.5.1 or greater.
```

Comment:
Is this note still accurate?

Reply:
Good point. Those are version that are over a year old. Removed.